src/core/client/admin/components/UserRole/SiteModeratorActionsModal.tsx

```diff
@@ -0,0 +1,193 @@
+import { Localized } from "@fluent/react/compat";
+import React, { FunctionComponent } from "react";
+import { Form } from "react-final-form";
+
+import useCommonTranslation, {
+  COMMON_TRANSLATION,
+} from "coral-admin/helpers/useCommonTranslation";
+
+import {
+  Button,
+  CallOut,
+  Card,
+  CardCloseButton,
+  Flex,
+  HorizontalGutter,
+  ListGroup,
+  ListGroupRow,
+  Modal,
+  Typography,
+} from "coral-ui/components/v2";
+
+import ModalBodyText from "../ModalBodyText";
+import ModalHeader from "../ModalHeader";
+import ModalHeaderUsername from "../ModalHeaderUsername";
+import SiteRoleActionsSites from "./SiteRoleActionsSites";
+
+import { SiteRoleActions_user as User } from "coral-admin/__generated__/SiteRoleActions_user.graphql";
+import { SiteRoleActions_viewer as Viewer } from "coral-admin/__generated__/SiteRoleActions_viewer.graphql";
+
+import styles from "./SiteRoleActionsModal.css";
+
+export interface Props {
+  open?: boolean;
+  onCancel: () => void;
+  onSubmit: (values: any) => void;
+  mode: "promote" | "demote" | null;
+  username: string | null;
+  siteRoleScopes: User["membershipScopes"] | User["moderationScopes"];
+  viewer: Viewer;
+}
+
+const SiteRoleActionsModal: FunctionComponent<Props> = ({
```

Comment:
Can you please rename this to `SiteModeratorActionsModal` as well to match the filename? And also the CSS file it's using?